calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
Oh thanks for pointing that out about the selectors vs labels.  I was kind of thinking of them as interchangeable but I think I get the differentiation.  Labels can be thought of as a key:value on a workload and a selector is matching one or more key:value for the purposes of applying policy.
Please correct me if I'm wrong.